### PR TITLE
docs(api.md): add note for page.setCookie method

### PR DESCRIPTION
**Pull request by ghost**
Knowing that `name` and `value` are required is nice, but this doesn't tell us everything we need to know.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -973,6 +973,8 @@ Shortcut for [page.mainFrame.select()](#frameselectselector-values)
   - `sameSite` <[string]> `"Strict"` or `"Lax"`.
 - returns: <[Promise]>
 
+> **NOTE** For each cookie object, at least one of `url` and `domain` must be specified.
```
Comment (aslushnikov) [inline docs/api.md:976]: @JoelEinbinder why both `domain` and `url` are needed? Can we fix this in protocol?

**Comment by JoelEinbinder** [comment]
Thanks for the PR!
By default, the current url of the page is set: See https://github.com/GoogleChrome/puppeteer/blob/master/lib/Page.js#L260

**Comment by ghost** [comment]
Oh, I didn't know that (and well, I'll document it too).
But do you know what happens when you use `page.setCookie` *before* `page.goto`?
...

😬

**Comment by JoelEinbinder** [comment]
It fails because `about:blank` can't have cookies. We should have a better error message in Puppeteer when you try to use setCookies on `about:blank` with no `url`. Are you interested in sending in a PR for that?

**Comment by ghost** [comment]
I could do that, yeah. In another PR?
There should still be a note for that in the doc, right?

**Comment by aslushnikov** [comment]
@JoelEinbinder shall we merge this?

**Comment by JoelEinbinder** [comment]
Closing this for now. Feel free to send another PR for the `about:blank` note.

**Pull request closed without merging**